src/cov.jl

```diff
@@ -45,6 +60,15 @@ if VERSION < v"0.5.0-dev+679"
     Base.cov(x::DenseMatrix, wv::WeightVec; mean=nothing, vardim::Int=1) =
         scale!(scattermat(x, wv; mean=mean, vardim=vardim), inv(sum(wv)))
 
+
+"""
+    mean_and_cov(x, [wv,] [vardim=1]) -> (Vector, Matrix)
```

Comment:
As I noted above, the PR hasn't been merged yet. Anyway, the Julia docs are large and take a lot of work to fix, so that's not necessarily the best reference.